appsec-kit-backend/src/main/java/com/vaadin/appsec/backend/AppSecScanEvent.java

```diff
@@ -20,6 +23,9 @@
  */
 public class AppSecScanEvent extends EventObject {
 
+    private static final Logger LOGGER = LoggerFactory
+            .getLogger(AppSecScanEvent.class);
+
```

Comment:
I would avoid having loggers in event objects, as they're purpose is to contain data not doing anything.